Makefile

```diff
@@ -107,12 +107,14 @@ generate-sanity-test:
 # Metalava & kotlin binary compatibility validator: check API
 .PHONY: check-api
 check-api:
+	./gradlew assembleRelease;
```

Comment:
nit: Can we make `./gradlew assembleRelease` as a `PHONY` target and use it. I have seen it is used in few places in the file.